frontend/lib/ui/ProgressList/ProgressList.module.css

```diff
@@ -47,7 +47,7 @@
     &:global(.updates) {
       aside {
         svg {
-          fill: var(--menu-updates);
+          fill: var(--link-hover);
```

Comment:
Not sure we should use `--link-hover` var here? Semantically, it would be better to keep the `--menu-updates` var here to control the color of this icon? Might want to rename it's name to be a bit more concise.

Reply:
I can keep 2 variables, pointing to the same color, if that is what we prefer?

Reply:
yeah think that would be wise here. Easier if we decide to change the color of the icon for some reason.

Reply:
Changed in 3daa4d9e0b473944f6053ce90993cd290237f796